cerberus/slack/slack_client.py

```diff
@@ -23,12 +25,24 @@ def initialize_slack_client():
 
 # Post messages and failures in slack
 def post_message_in_slack(slack_message, thread_ts=None):
-    slack_client.chat_postMessage(
-        channel=slack_channel_ID,
-        link_names=True,
-        text=slack_message,
-        thread_ts=thread_ts
-    )
+    max_retries = 10
```

Comment:
I don't really know the slack code/threading, but could we batch messages for slack and send no more than often than every minute?  I worried we will end up spinning a lot of threads that are spinning retrying sends.   Maybe have a queue of messages for slack that we drain every 63 seconds.